### PR TITLE
Add ui component plugin extension point

### PR DESCRIPTION
**Pull request by barmac**

### CODE REVIEW

**Comment by nikku** [comment]
@barmac I've added the ability to ship core plugins with the app and performed some other minor adjustments: https://github.com/camunda/camunda-modeler/pull/1490/commits/7391ab2c0f92a9dcd89bb4558771feea692d4c8e

Looks good to me for now.

**Comment by nikku** [comment]
You could port / build the deployment tool on top of the core plug-in stub I've checked in.

File: client/src/plugins/deployment-tool/DeploymentTool.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { Fill } from '../../app/slot-fill';
+
+export default class DeploymentTool extends PureComponent {
+
+  render() {
+
+    return (
+      <React.Fragment>
+        <Fill name="toolbar" group="extension">
+          🤩
```
Comment (barmac) [inline client/src/plugins/deployment-tool/DeploymentTool.js]: 🤩

**Comment by barmac** [comment]
Cool changes. I'll port the deployment tool plugin branch on top of `develop`.

**Pull request merged**